app/server/models/credit_transfer.py

```diff
@@ -380,3 +401,14 @@ def __init__(self,
 
         self.append_organisation_if_required(self.recipient_transfer_account.organisation)
         self.append_organisation_if_required(self.sender_transfer_account.organisation)
+
+        if require_sufficient_balance and not self.check_sender_has_sufficient_balance():
```

Comment:
just check a credit transfer having sufficient balance by default, since it's kinda the norm